x-pack/test/alerting_api_integration/common/plugins/alerting/server/routes.ts

```diff
@@ -875,45 +875,6 @@ export function defineRoutes(
   );
 
   // Remove when we have real routes
-  router.post(
-    {
-      path: '/api/alerts_fixture/preview_alert_deletion',
-      security: {
-        authz: {
-          enabled: false,
-          reason: 'This route is opted out from authorization',
-        },
-      },
-      validate: {
-        body: schema.object({
-          isActiveAlertDeleteEnabled: schema.boolean(),
-          isInactiveAlertDeleteEnabled: schema.boolean(),
-          activeAlertDeleteThreshold: schema.number({ min: 1 }),
-          inactiveAlertDeleteThreshold: schema.number({ min: 1 }),
-          categoryIds: schema.maybe(schema.arrayOf(schema.string())),
-        }),
-      },
-    },
-    async (
-      context: RequestHandlerContext,
-      req: KibanaRequest<any, any, any, any>,
-      res: KibanaResponseFactory
-    ): Promise<IKibanaResponse<any>> => {
-      const [, { spaces }] = await core.getStartServices();
-      const spaceId = spaces ? spaces.spacesService.getSpaceId(req) : 'default';
-      const alerting = await alertingStart;
-
-      try {
-        const result = await alerting.previewAlertDeletion(req.body, spaceId);
```

Comment:
we can remove `previewAlertDeletion` from the alerting plugin start function as well

Reply:
done in https://github.com/elastic/kibana/pull/216504/commits/d87ef93d9a0ad118fd552e15c743cf2b1d15d8b0